app/controllers/bookingsync_portal/admin/rentals_controller.rb

```diff
@@ -5,19 +5,44 @@ class RentalsController < Admin::BaseController
       before_action :fetch_remote_rentals, only: :index
 
       def index
-        @not_connected_rentals = current_account.rentals.visible.ordered.not_connected
-        @visible_rentals = current_account.rentals.visible
-        @remote_accounts = current_account.remote_accounts
-        @remote_rentals_by_account = current_account.remote_rentals.ordered
-          .includes(:remote_account, :rental).group_by(&:remote_account)
+        render :index, locals: { **index_arguments }
       end
 
       def show
         rental
       end
 
+      def index_arguments
+        BookingsyncPortal.custom_arguments.dig("Admin", "RentalsController", "index") || {
```

Comment:
this looks quite static, whole operation should be customizable

Reply:
if you want to customize all operation, wouldn't be better to override the action?

Reply:
It would be but it's not easy to override just one action from controller, and even more painful to test it, so the point of this operation is to give a mean to do that without all the overhead